src/scss/mixins/variables/_animation.scss

```diff
@@ -0,0 +1,10 @@
+$animation-time: 250ms !default;
+
+@mixin componentAnimationVars() {
+  /* Component animation variables */
```

Comment:
Nitpick: IMO the vars are expressive enough to not need these comments.

Reply:
It's just a nice separator in the output though. I'll leave it for now.